shared/constants/network.ts

```diff
@@ -376,7 +378,7 @@ export const LINEA_SEPOLIA_TOKEN_IMAGE_URL = './images/linea-logo-testnet.png';
 export const LINEA_MAINNET_TOKEN_IMAGE_URL = './images/linea-logo-mainnet.svg';
 export const TEST_ETH_TOKEN_IMAGE_URL = './images/black-eth-logo.svg';
 export const BNB_TOKEN_IMAGE_URL = './images/bnb.svg';
-export const MATIC_TOKEN_IMAGE_URL = './images/matic-token.svg';
```

Comment:
I'm still seeing various references to this image in test files and Storybook, and the image itself remains present. Should we we remove the image and the remaining references? It would be nice to remove the extra image for the sake of reducing bundle size.

Reply:
Sure I can remap these references.

Reply:
https://github.com/MetaMask/metamask-extension/pull/26671/commits/d75c312097bb719a6a0dad166d775636ab5c716a